src/Creator.tsx

```diff
@@ -120,63 +113,35 @@ export const Creator = ({ NFToupon_Key }: Props) => {
   };
 
   const acceptOffer = async () => {
-    const response = await fetch(
-      `https://eatozee-crypto.app/api/nftoupon/offer/accept`,
-      {
-        method: 'POST',
-        headers: {
-          'Content-Type': 'application/json',
-          'NFToupon-Key': NFToupon_Key,
-        },
-        body: JSON.stringify({
-          tokenOfferIndex: details.tokenOfferIndex,
-          address: walletAddress,
-        }),
-      }
-    );
-    const { payload } = await response.json();
+    const payload = {
+      tokenOfferIndex: details.tokenOfferIndex,
+      address: walletAddress,
+    };
+    const result = await fetcher(NFToupon_Key, ACCEPT_OFFER_URL, payload);
```

Comment:
the response should return the payload object ... could you please double check...